src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.Properties.cs

```diff
@@ -63,7 +63,11 @@ private void WritePropertyDefinition(IPropertyDefinition property)
             if (property.GetHiddenBaseProperty(_filter) != Dummy.Property)
                 WriteKeyword("new");
 
+            if (property.ReturnValueIsByRef)
+                WriteKeyword("ref");
```

Comment:
I couldn't find the C# spec but is ref supposed to be the thing directly in front of the type?